deploy.sh

```diff
@@ -429,6 +431,7 @@ function helm_region_shared {
     --set-string "oauth2_proxy.client_secret=${CLOUD_ROBOTICS_OAUTH2_CLIENT_SECRET}"
     --set-string "oauth2_proxy.cookie_secret=${CLOUD_ROBOTICS_COOKIE_SECRET}"
     --set "use_tv_verbose=${CRC_USE_TV_VERBOSE}"
+    --set "secret_manager_plugin=${GKE_SECRET_MANAGER_PLUGIN}"
```

Comment:
This line is not needed/useful if you don't add it to values.yaml for the base cloud chart.

Reply:
+1. But telling charts whether it is enabled or not might be useful though.